arch/x86/kernel/head_64.S

```diff
@@ -528,11 +528,11 @@ NEXT_PAGE(zero_pud)
 	FILL(zero_pmd - __START_KERNEL_map + __PAGE_KERNEL_RO, 512)
 
 NEXT_PAGE(poisoned_pte)
-	FILL(poisoned_page - __START_KERNEL_map + __PAGE_KERNEL_RO, 512)
+	FILL(poisoned_page - __START_KERNEL_map + _KERNPG_TABLE, 512)
```

Comment:
The page is not poisoned anymore. Please rename s/poisoned/kasan_early/ or something along these lines.